src/client/datascience/commands/commandRegistry.ts

```diff
@@ -552,3 +562,26 @@ export class CommandRegistry implements IDisposable {
         }
     }
 }
+
+export function getImportCodeForFileType(filepath: string) {
```

Comment:
Seems like a weird spot for this? Not sure what it has to do with the commandRegistry?